Snowcast/ResortsTableViewController.swift

```diff
@@ -30,17 +48,25 @@ class ResortsTableViewController: UITableViewController {
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath)
+        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath) as! ResortTableViewCell
 
-        cell.textLabel?.text = self.resorts[indexPath.row].name
+        cell.resortNameLabel.text = self.resorts[indexPath.row].name
 
         return cell
     }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return 80
+    }
+    
+    override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        self.performSegueWithIdentifier("resortDetailSegue", sender: tableView)
+    }
 
     // MARK: - Navigation
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
-        if (segue.identifier == "showResortDetailSegue") {
+        if (segue.identifier == "resortDetailSegue") {
```

Comment:
Control Statement Parentheses Violation: if statements shouldn't wrap their conditionals in parentheses.